scripts/build_sc_component_manager.sh

```diff
@@ -6,7 +6,4 @@ then
   source "$(cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd)"/../sc-component-manager/scripts/set_vars.sh
 fi
 
-# Need to define this variables to execute build_sc_component_manager.sh
-export ROOT_CMAKE_PATH="${SC_COMPONENT_MANAGER_PATH}"
```

Comment:
This script will not build sc-component manager with these changes. It is needed to think about that we can build sc-machine and sc-component-manager separately, but without envs force changes